package.json

```diff
@@ -13,35 +13,35 @@
   },
   "dependencies": {
     "@tailwindcss/typography": "^0.5.8",
-    "@types/gtag.js": "^0.0.12",
-    "@types/marked": "^4.0.8",
+    "@types/gtag.js": "^0.0.14",
+    "@types/marked": "^5.0.2",
     "@types/prismjs": "^1.16.6",
     "ansi-regex": "6.0.1",
     "babel-plugin-prismjs": "^2.1.0",
     "gray-matter": "^4.0.3",
-    "marked": "^4.2.5",
-    "next": "12.1.5",
+    "marked": "^9.0.3",
+    "next": "13.5.3",
     "prismjs": "^1.29.0",
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "react-icons": "^4.7.1",
     "react-share": "^4.4.1"
   },
   "devDependencies": {
-    "@testing-library/jest-dom": "^5.16.4",
-    "@testing-library/react": "^13.2.0",
-    "@types/node": "^17.0.25",
-    "@types/react": "18.0.6",
+    "@testing-library/jest-dom": "^6.1.3",
+    "@testing-library/react": "^14.0.0",
+    "@types/node": "^20.8.0",
+    "@types/react": "18.2.24",
     "autoprefixer": "^10.4.5",
-    "eslint": "8.14.0",
-    "eslint-config-next": "12.1.5",
-    "eslint-config-prettier": "^8.5.0",
-    "jest": "^28.1.0",
-    "jest-environment-jsdom": "^28.1.0",
-    "next-sitemap": "^2.5.20",
+    "eslint": "8.50.0",
+    "eslint-config-next": "13.5.3",
+    "eslint-config-prettier": "^9.0.0",
+    "jest": "^29.7.0",
+    "jest-environment-jsdom": "^29.7.0",
+    "next-sitemap": "^4.2.3",
     "postcss": "^8.4.12",
-    "prettier": "^2.6.2",
+    "prettier": "^3.0.3",
```

Comment:
https://github.com/prettier/prettier-vscode/issues/3007

This issue occurs but restarting vscode is solution